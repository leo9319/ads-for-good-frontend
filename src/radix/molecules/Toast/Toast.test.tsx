import React from 'react';
import { render } from '@testing-library/react';
import ToastMolecule from './Toast';

describe('ToastMolecule', () => {
  it('should render successfully', () => {
    const { baseElement } = render(
      <ToastMolecule
        iconName="icon-info"
        alternateCloseIcon="icon-close"
        title="We're excited to see you again!"
        type="info"
        isVisible
        xPosition="center"
        yPosition="top"
      >
        Access your personalized dashboard, manage your preferences, and explore
        all the features we have to offer.
      </ToastMolecule>
    );
    expect(baseElement).toMatchSnapshot();
  });
});
