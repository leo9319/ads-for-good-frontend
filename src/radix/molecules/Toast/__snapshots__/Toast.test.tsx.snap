// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastMolecule should render successfully 1`] = `
<body>
  <div>
    <div
      class="toast toast-x-center toast-y-top"
    >
      <div
        class="rt-CalloutRoot rt-r-size-2 rt-variant-soft rt-Flex rt-r-ai-stretch rt-r-jc-center calloutRoot info toast_inner isToast"
        data-accent-color=""
      >
        <div
          class="rt-Box callout"
        >
          <div
            class="rt-Box contentbox"
          >
            <p
              class="rt-Text rt-r-size-2 rt-CalloutText title body-md typography-font-weight-semibold"
            >
              We're excited to see you again!
            </p>
            <div
              class="rt-Box body-sm typography-font-weight-regular"
            >
              Access your personalized dashboard, manage your preferences, and explore all the features we have to offer.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
